From: AMEET M. PARANJAPE <aparanja@redhat.com>
Date: Tue, 5 May 2009 08:52:53 -0500
Subject: [ppc64] adjust oprofile_cpu_type detail
Message-id: 4A0044B5.5050400@REDHAT.COM
O-Subject: Re: [PATCH RHEL5.4 BZ496709] PPC64: adjust oprofile_cpu_type
Bugzilla: 496709
RH-Acked-by: David Howells <dhowells@redhat.com>

David Howells wrote:
>> Anyway, here is a new patch that uses compat-v1 for oprofile_cpu_type instead
>> of compat-<arch>.
>

A portion of this patch was reworked because the community wanted more detail
in the compat string.  The patch has been accepted into the powerpc tree:
79af6c49a93ea80952fd42dc180c8827cd8c984c

The brewbuild built fine and is here:
https://brewweb.devel.redhat.com/taskinfo?taskID=1778756

--
Ameet M. Paranjape
aparanja@redhat.com
IBM/Red Hat POWER Liason
IRC name: aparanja

diff --git a/arch/powerpc/kernel/cputable.c b/arch/powerpc/kernel/cputable.c
index 4d0e934..179ea44 100644
--- a/arch/powerpc/kernel/cputable.c
+++ b/arch/powerpc/kernel/cputable.c
@@ -275,7 +275,7 @@ static struct cpu_spec cpu_specs[] = {
 		.cpu_user_features	= COMMON_USER_POWER5_PLUS,
 		.icache_bsize		= 128,
 		.dcache_bsize		= 128,
-		.oprofile_cpu_type      = "ppc64/compat-v1",
+		.oprofile_cpu_type      = "ppc64/ibm-compat-v1",
 		.oprofile_type		= PPC_OPROFILE_POWER4,
 		.platform		= "power5+",
 	},
@@ -320,7 +320,7 @@ static struct cpu_spec cpu_specs[] = {
 		.cpu_user_features	= COMMON_USER_POWER6,
 		.icache_bsize		= 128,
 		.dcache_bsize		= 128,
-		.oprofile_cpu_type      = "ppc64/compat-v1",
+		.oprofile_cpu_type      = "ppc64/ibm-compat-v1",
 		.oprofile_type		= PPC_OPROFILE_POWER4,
 		.platform		= "power6",
 	},
